### ISSUE TITLE
Custom Mesh

### ISSUE THREAD

**Issue by Areiban** [marcomusy/vedo#219]
Hello, I have a problem to create a custom Mesh similar to image
![Figure_1](https://user-images.githubusercontent.com/28365704/94297838-c6f6e500-ff2a-11ea-956e-1620f3872fc3.png)

When using the Mesh ([verts, faces]) function, it causes problems when filling the region delimited by the vertices of the mesh; What would be the proper function for such a purpose?

This is the vertices list
vertices1 = [(10, 10, 10), (30, 30, 10), (20, 50, 10), (40, 40, 10), (60, 50, 10), (50, 30, 10), (70, 10, 10), (45, 0, 10)]
vertices2 = [(10, 10, 20), (30, 30, 20), (20, 50, 20), (40, 40, 20), (60, 50, 20), (50, 30, 20), (70, 10, 20), (45, 0, 20)]

Basically I want to build an irregular polygon and then join the top and bottom faces to form a solid

Thanks

**Comment by marcomusy** [comment]
Hi Adrian, try:
```python

from vedo import *
verts = [(10, 10, 10), (30, 30, 10), (20, 50, 10), (40, 40, 10), (60, 50, 10), (50, 30, 10), (70, 10, 10), (45, 0, 10)]
ln = Line(verts, closed=True)
polyg = ln.clone().triangulate() # clone just makes a copy of ln
extruded = polyg.extrude(10)
show(ln, polyg, extruded, N=3, axes=1)
```
![image](https://user-images.githubusercontent.com/32848391/94301102-58d21280-ff6a-11ea-848f-74ffce2eb796.png)

**Comment by Areiban** [comment]
Thank you very much for the help =)
